packages/driver/src/cypress/error_messages.ts

```diff
@@ -1665,6 +1665,32 @@ export default {
     },
   },
 
+  switchToDomain: {
+    experiment_not_enabled: {
+      message: `${cmd('switchToDomain')} requires enabling the experimentalMultiDomain flag`,
+      docsUrl: 'https://on.cypress.io/switchToDomain',
+    },
+    invalid_domain_argument: {
+      message: `${cmd('switchToDomain')} requires the first argument to be a string. You passed: \`{{arg}}\``,
+      docsUrl: 'https://on.cypress.io/switchToDomain',
+
+    },
+    invalid_fn_argument: {
+      message: `${cmd('switchToDomain')} requires the last argument to be a function. You passed: \`{{arg}}\``,
+      docsUrl: 'https://on.cypress.io/switchToDomain',
+
+    },
+    run_domain_fn_errored: {
+      message: stripIndent`
+      ${cmd('switchToDomain')} failed with the following error:
+
+        > {{error}}
+
+      This is likely because the data argument specified is not serializable. Note that functions and DOM objects cannot be serialized.`,
```

Comment:
Do we need additional guidance for things like variable scope issues with the stringify'd function or is that better suited for API doc?

Reply:
That type of error isn't handled in this PR, but it is in #19465. Since it's handled separately, it will have a different error message than this one, but it's a good idea to add something about variable scope issues, so I'll make sure to bring that up for #19465. Good call.